ark/segmentation/marker_quantification.py

```diff
@@ -27,6 +29,10 @@ def compute_marker_counts(input_images, segmentation_labels, nuclear_counts=Fals
             morphology features for regionprops to extract for each cell
         split_large_nuclei (bool):
             controls whether nuclei which have portions outside of the cell will get relabeled
+        extraction (str):
```

Comment:
Can you change this to have a more informative name instead of default? Maybe pixel_sum, linear_sum, unweighted_sum, etc